Source/WebCore/rendering/RenderElement.cpp

```diff
@@ -1833,6 +1833,11 @@ const RenderStyle* RenderElement::targetTextPseudoStyle() const
 
 bool RenderElement::getLeadingCorner(FloatPoint& point, bool& insideFixed) const
 {
+    if (isSVGRenderer()) {
```

Comment:
Any time code in RenderFoo consults its own type, you should consider making functions virtual and using polymorphism. But in this case the next clause does something similar, so this isn't adding too much mess.

Reply:
Noted.